scripts/package.json

```diff
@@ -78,5 +76,6 @@
     "webpack-dev-server": "3.1.14",
     "webpack-notifier": "^1.6.0",
     "yargs": "^6.5.0"
-  }
+  },
+  "main": "./index.js"
```

Comment:
Nitpick but could you move this earlier in the file, where it tends to be in other files?

Reply:
Again used "sort-package-json" for this. It's like a prettier for the package.json sort order. I'm going to probably put that in a precommit step...